pkg/constants/constants.go

```diff
@@ -156,6 +156,9 @@ const (
 
 	// MetricsAnnotation is the annotation used for enabling/disabling metrics
 	MetricsAnnotation = "openservicemesh.io/metrics"
+
+	// OutboundPortExclusionListAnnotation is the annotation used for outbound port exclusion
+	OutboundPortExclusionListAnnotation = "openservicemesh.io/outbound-port-exclusion-list"
```

Comment:
Let's define this within the `injector` package in the same file where it is used since this doesn't need to be global at the moment.